### ISSUE TITLE
Flogo Web UI doesn't create an executable binary on macOS

### ISSUE THREAD

**Issue by retgits** [TIBCOSoftware/flogo#308]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[] feature request
[] support request
[] general question
```

**Current behavior (how does the issue manifest):** When you build your app using the Flogo Web UI, the app misses the execution bit and can only be run after executing `chmod +x <app>`

**Expected behavior:** After the binary is finished downloading I want to run my app without requiring extra steps

**Minimal steps to reproduce the problem (not required if feature enhancement):**

* Build an app using the Web UI
* run `./<app>`

**Please tell us about your environment (Operating system, docker version, browser & web ui version, etc):** Flogo Web UI 0.5.4

**Flogo version (CLI & contrib/lib. If unknown, leave empty or state unknown):** 

```
flogo cli version [v0.5.5]
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by fcastill** [comment]
I don't think we can keep the executable permissions unless we tar/zip the binary before triggering the download.
